llvm/lib/Target/AMDGPU/AsmParser/AMDGPUAsmParser.cpp

```diff
@@ -3166,6 +3166,13 @@ AMDGPUAsmParser::parseRegister(bool RestoreOnFailure) {
   return AMDGPUOperand::CreateReg(this, Reg, StartLoc, EndLoc);
 }
 
+static bool isInv2PiToken(const AsmToken &Tok) {
+  if (!Tok.is(AsmToken::Identifier))
+    return false;
+  StringRef Str = Tok.getIdentifier();
+  return (Str.str() == "INV2PI" || Str.str() == "INV2PI64");
```

Comment:
```suggestion
  return Str == "INV2PI" || Str. == "INV2PI64";
```

Don't need all these conversions to std::string